scripts/upgrade_kernel.sh

```diff
@@ -24,4 +24,10 @@ sudo grubby \
   --update-kernel=ALL \
   --args="psi=1"
 
+# use the tsc clocksource by default
+# https://repost.aws/knowledge-center/manage-ec2-linux-clock-source
+sudo grubby \
+  --update-kernel=ALL \
+  --args="clocksource=tsc tsc=reliable"
+
```

Comment:
This isn't strictly necessary with the `configure-clocksource` unit in place, but the `tsc=reliable` flag is still worth setting, and we might as well configure a default here too.